src/app/modules/map-config/components/visualisations/visualisations.component.html

```diff
@@ -7,31 +7,34 @@
     </div>
 </div>
 
-<table mat-table [dataSource]="visualisationsFa.value" class="mat-elevation-z1">
+<table mat-table [dataSource]="visualisationsFa.value" class="mat-elevation-z1" cdkDropList
+(cdkDropListDropped)="drop($event)">
 
     <ng-container matColumnDef="name">
-        <th mat-header-cell *matHeaderCellDef>{{'Name' | translate}}</th>
-        <td mat-cell *matCellDef="let element"> {{element.name}} </td>
+        <th class="th-name" mat-header-cell *matHeaderCellDef>{{'Name' | translate}}</th>
+        <td class="th-name" mat-cell *matCellDef="let element"> {{element.name}} 
+            <mat-icon cdkDragHandle class="td-draghandle" >drag_indicator</mat-icon>
+        </td>
     </ng-container>
 
     <ng-container matColumnDef="layers">
-        <th style="width: 60%;" mat-header-cell *matHeaderCellDef>{{'Layers' | translate}}</th>
-        <td mat-cell *matCellDef="let element">
+        <th class="th-layers" mat-header-cell *matHeaderCellDef>{{'Layers' | translate}}</th>
+        <td class="th-layers" mat-cell *matCellDef="let element">
             <div style="display: flex;">
                 <span style="display: inline-block; margin-right: 5px;" *ngFor="let l of element.layers"> • {{l}}</span>
             </div>
         </td>
     </ng-container>
 
     <ng-container matColumnDef="displayed">
-        <th style="width: 8%; text-align: center;" mat-header-cell *matHeaderCellDef>{{'Displayed by default' | translate}}</th>
-        <td style="text-align: center;" mat-cell *matCellDef="let element"> 
+        <th class="th-display" mat-header-cell *matHeaderCellDef>{{'Displayed by default' | translate}}</th>
+        <td class="th-display" mat-cell *matCellDef="let element"> 
             <mat-checkbox [checked]="element.displayed" [disabled]="true"></mat-checkbox>
         </td>
     </ng-container>
     <ng-container matColumnDef="action">
-        <th mat-header-cell *matHeaderCellDef>{{'Actions' | translate}}</th>
-        <td mat-cell *matCellDef="let element">
+        <th class="th-action" mat-header-cell *matHeaderCellDef>{{'Actions' | translate}}</th>
+        <td class="th-action" mat-cell *matCellDef="let element">
```

Comment:
`class="th-action"` => on a `td` element is it normal ?

Reply:
fixed